persistence/trees/trees.go

```diff
@@ -1,7 +1,17 @@
 // package trees maintains a set of sparse merkle trees
-// each backed by the KVStore interface. It offers an atomic
+// each backed by the `KVStore` interface. It offers an atomic
 // commit and rollback mechanism for interacting with
 // that core resource map of merkle trees.
```

Comment:
"that core resource" doesnt make sense to me

Reply:
Reworded it to make it clearer 👍